### ISSUE TITLE
[Bug]: Gaining zero of a resource makes the +X number red in the notification

### ISSUE THREAD

**Issue by BenLubar** [MelvorIdle/melvoridle.github.io#3749]
### Describe the bug

Fight a low-level monster. If your first increase in experience for a skill is 0, the notification's number will be invisible until you gain experience, after which it will be red instead of green.

This does not occur if you *lose* zero of a resource. For example, killing a fresh easy slayer task before the notification expires results in a new notification separate from the notification that just has a slayer coin icon in it and no text.

Mods (none of these affect notifications)
```
[Myth] Combat Simulator
SEMI Core
SEMI Auto Master
SEMI Drop Chances
Semi Auto Township
SEMI Auto Bury
Tiny Passive Icons
Show Slayer Task Attack Style
AutoSort
Shining Star
Item Uses
ETA
QuickShards
SEMI Auto Bonfire
```

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [X] Steam
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Mobile App (Android Beta)
- [ ] Other (Please Specify)

### Which expansions are purchased and active at the time of the bug?

- [X] Throne of the Herald
- [X] Atlas of Discovery

### Which version of the game are you experiencing this bug on?

v1.2.1

### Game Subversion

?7937

### Are you using any mods?

Yes

**Comment by Sycamore5** [comment]
I’ve noticed the red color too, and I don’t have any mods.

**Comment by MelvorIdle** [comment]
Fixed with Game Update - v1.2.1 ?7958
